core/src/main/scala/kafka/controller/ControllerContext.scala

```diff
@@ -83,6 +83,8 @@ class ControllerContext {
   var epochZkVersion: Int = KafkaController.InitialControllerEpochZkVersion
 
   val allTopics = mutable.Set.empty[String]
+  var topicIds = mutable.Map.empty[String, UUID]
```

Comment:
Should these 2 names be more self-explanatory?

Reply:
What part of the names is unclear? The name describes values of the map, so is it not clear what the keys should be?

Reply:
Well, `topicIds` and `topicNames` may already enough self-explanatory.